_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/069af6d6f3294421ec163b07fef91e5d?security_tenant=global -a basic -c admin:1234 -e ses -s user@amazon.com -r user@amazon.com -f png
+```
+
+### Example: Request a PDF file
+
+The following command requests a PDF file and specifies the email address for the recipient.
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:Test@1234 -e ses -s user@domain.com -r user@domain.com -f pdf
+```
+
+Upon success, the file will be sent to the specified email address.
+
+![PDF example]({{site.url}}{{site.baseurl}}/images/dashboards/cli-pdf-report.png)
+
+
+### Example: Request a CSV file
+
+The following command generates a report that contains all table content in CSV format, and sends the report using SES transport to an email address.
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/f166434919d2d8d3708a4705b6f2ed2d?security_tenant=global -f csv -a basic -c admin:Test1234 -e ses -s user@amazon.com -r user@amazon.com
+```
+
+Upon success, you will receive the email with the CSV file attached.
+
+<!--Question for eng: Is the rest of the command similar to the other examples with exception of "-a cognito" specified for authentication? If so, perhaps we don't need this example.
+### Example: Request a report with Cognito authentication
+
+The following command generates a report for download to the directory that you run the CLI command.
+
+```
+opensearch-reporting-cli 
+```
+
+Upon success, you will see "Report downloaded" in the command line.
+-->
+
+## Get help
+
+To get a list of all available CLI arguments, run the following command.
+
+``` 
+$ opensearch-reporting-cli -h
+```
+
+
+## Reporting CLI options
+
+You can use any of the following arguments with the `opensearch-reporting-cli` tool.
+
+| Argument      | Description       | Acceptable values and usage | Environment variable
+:--------------------- | :--- | :--- |
+`-u`, `--url` | URL for the Dashboard visualization. | Obtain from OpenSearch Dashboards > Visualize > Share > Permalinks > Copy link. | URL
+`-a`, `--auth` | Authentication type for the report. | You can specify either Basic `basic`, Cognito `cognito`, SAML `saml` or No Auth `none`. If no value is specified, it defaults to No auth. Cognito and SAML require credentials with `-c` flag. | N/A
+`-c`, `--credentials` | OpenSearch login credentials. | Enter username and password separated by a colon. For example, username:password. Required for Cognito and SAML auth types. | USERNAME and PASSWORD
+`-t`, `--tenant` | Tenants in OpenSearch dashboards. | The default tenant is global.| N/A
```

Comment:
I recently noticed `private` is default tenant in website so I changed default tenant to private in this [PR](https://github.com/opensearch-project/reporting-cli/pull/5/files#diff-2237fab1cd95514388b409d01aedefad56e149e4a27b50969d32a4a1da6a6cf5)